src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/QualificationValidator.cs

```diff
@@ -44,6 +44,16 @@ public QualificationValidator(long ruleId, IList<string> qualifications)
                     .WithErrorCode("6")
                 .WithRuleId(ruleId);
 
+            When(x => x.QualificationType != null && x.QualificationType.Contains("GCSE"), () =>

```

Comment:
There is already a check for qualification type not being empty at the top of the file. Shouldn't need the !- null check